azext_iot/operations/events3/_builders.py

```diff
@@ -0,0 +1,143 @@
+import asyncio
+import uamqp
+from time import time
+
+from azext_iot.common.sas_token_auth import SasTokenAuthentication
+from azext_iot.common.utility import (parse_entity, unicode_binary_map, url_encode_str)
+
+DEBUG = True
+
+class EventTargetBuilder():
+
+    def buildIotHubTargetSync(self, target):
+        eventLoop = asyncio.new_event_loop()
```

Comment:
You can put shared initialization in a `def __init__(self):` method for the class. This new event loop logic would make sense to go there.

Reply:
Updated so that eventloop logic is in __init__